packages/accordions/src/styled/accordion/StyledAccordion.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'accordions.accordion';
+
+export const StyledAccordion = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
```

Comment:
Will it help consumers down the line if you set `display: flex;` on this outer element?

Reply:
This element could be set to `display: flex; flex-direction: column;` but I'm unaware if this helps consumers down the line. Do you think it would be helpful? I can add it if it does 👍 .

Reply:
If the world around the accordion is `flex`, then this could be a natural benefit as the flex parent can make helpful assertions about the layout of the child. But I'll let you apply the imaginative scenario stress to determine whether this ends up providing helpful value or not.

Reply:
I ran through a couple of scenarios and concluded to leave `flex` off the outer element for the time being.